src/main/java/com/hivemq/cli/commands/hivemq/script/ScriptCreateCommand.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2019-present HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.cli.commands.hivemq.script;
+
+import com.hivemq.cli.MqttCLIMain;
+import com.hivemq.cli.commands.hivemq.datahub.DataHubOptions;
+import com.hivemq.cli.commands.hivemq.datahub.OutputFormatter;
+import com.hivemq.cli.commands.hivemq.datahub.ScriptDefinitionOptions;
+import com.hivemq.cli.converters.ScriptTypeConverter;
+import com.hivemq.cli.hivemq.scripts.CreateScriptTask;
+import com.hivemq.cli.openapi.hivemq.DataHubScriptsApi;
+import com.hivemq.cli.rest.HiveMQRestService;
+import org.jetbrains.annotations.NotNull;
+import org.tinylog.Logger;
+import picocli.CommandLine;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.Callable;
+
+@CommandLine.Command(name = "create",
+                     description = "Create a new script",
+                     synopsisHeading = "%n@|bold Usage:|@  ",
+                     descriptionHeading = "%n",
+                     optionListHeading = "%n@|bold Options:|@%n",
+                     commandListHeading = "%n@|bold Commands:|@%n",
+                     versionProvider = MqttCLIMain.CLIVersionProvider.class,
+                     mixinStandardHelpOptions = true)
+public class ScriptCreateCommand implements Callable<Integer> {
+
+    @SuppressWarnings({"NotNullFieldNotInitialized", "unused"})
+    @CommandLine.Option(names = {"-i", "--id"}, required = true, description = "The id of the script")
+    private @NotNull String scriptId;
+
+    @SuppressWarnings({"NotNullFieldNotInitialized", "unused"})
+    @CommandLine.Option(names = {"--type"},
+                        required = true,
+                        converter = ScriptTypeConverter.class,
```

Comment:
Type can be an enumeration, right? If there is already a converter in place we can use a safe type instead of a String.